samples/bluetooth/broadcast_audio_source/src/main.c

```diff
@@ -241,28 +323,62 @@ int main(void)
 	struct bt_le_ext_adv *adv;
 	int err;
 
+#if defined(CONFIG_SOC_NRF5340_CPUAPP)
+	/* Set nRF5340 cpu app core to 128 MHz */
+	err = nrfx_clock_divider_set(NRF_CLOCK_DOMAIN_HFCLK, NRF_CLOCK_HFCLK_DIV_1);
+
+	if (err != NRFX_ERROR_BASE_NUM) {
+		printk("Error changing clock to 128 MHz\n");
+	} else {
+		nrfx_clock_hfclk_start();
+		while (!nrfx_clock_hfclk_is_running()) {
+		}
```

Comment:
Do we need some kind of watchdog for this? Otherwise we may end in forever loop before `bt_enable` without any logging or output